checkbox.scss

```diff
@@ -96,7 +96,7 @@ $checkbox-active-color: unquote("rgba(#{$color-active}, 1.0)") !default;
   margin: 0;
   overflow: hidden;
   position: absolute;
-  top: 2px;
+  top: 1px;
```

Comment:
@shane-tomlinson I dont know if its just my screen, but the text next to the checkbox looks a *little* to the bottom:
### At 1px:
<img width="648" alt="screenshot 2016-06-16 16 05 34" src="https://cloud.githubusercontent.com/assets/432707/16121750/6016bb80-33dc-11e6-8f35-c43af206642d.png">

### At 1.5px:
<img width="635" alt="screenshot 2016-06-16 16 05 26" src="https://cloud.githubusercontent.com/assets/432707/16121770/6f051650-33dc-11e6-82e7-4168e3ada6c8.png">

I think its super-minute difference and wont be noticed by any of our users, but I feel that 1.5px seems more centered.